src/AuthMSAL/AuthMSAL.js

```diff
@@ -69,6 +69,12 @@ async function _acquireTokensByRequestAndAccount(tokenReq, account) {
       if (silentTokenFetchError.errorCode === 'no_tokens_found') {
         // No token found during acquireTokenSilent, ignore this error, nothing to do
         return;
+      } else if (silentTokenFetchError.errorCode === 'login_required') {
+        console.warn(
+          'Silent authentication not possible, user is not logged in. This is usually caused when the user session ' +
```

Comment:
Just a suggestion for phrasing: _This usually happens when_, I found _caused when_ a bit misformulated :slightly_smiling_face:

Reply:
Fixed, thank you for the feedback :slightly_smiling_face: